.golangci.yaml

```diff
@@ -196,10 +196,12 @@ issues:
   exclude-rules:
     - path: internal[/\\]app[/\\]backendapi[/\\]
       linters: [goimports, gochecknoglobals, errcheck, misspell, govet, unparam, golint, gofmt]
-    - path: internal[/\\]app[/\\]backendapi[/\\]
+    - path: internal[/\\]app[/\\]frontendapi[/\\]
```

Comment:
These rules were set up to exclude lint checking the legacy codes from v.5. #416 will disable all these rules from now on.